examples/fabric/nested_ingestion_pipeline/README.md

```diff
@@ -0,0 +1,96 @@
+# Nested pipelines
+
+The following example, exemplifies the utilization of nested pipelines including control activities like ForEach and If.
+
+![Nested Pipelines](pl_main_pipeline.png)
+
+The main pipeline is composed of 2 top activities:
+
+- a lookup activity named "Read Configuration File
+- a ForEach activity names "ForEach"
+
+Within the ForEach activity, there is a nested If activity called "If New Or Updated" and if this activity evaluates to True, a Invoke Pipeline activity named "Invoke Ingestion Pipeline" is triggered.
```

Comment:
```suggestion
Within the ForEach activity, there is a nested If activity called "If New Or Updated" and if this activity evaluates to True, an Invoke Pipeline activity named "Invoke Ingestion Pipeline" is triggered.
```

Reply:
Decided to simplify the sentence. Please check if it is OK now!